depends/packages/librustzcash.mk

```diff
@@ -3,8 +3,8 @@ $(package)_version=0.1
 $(package)_download_path=https://github.com/zcash/$(package)/archive/
 $(package)_file_name=$(package)-$($(package)_git_commit).tar.gz
 $(package)_download_file=$($(package)_git_commit).tar.gz
-$(package)_sha256_hash=b96a0646d4c4856bc6171dc26cce10644a6129ac92b73a91f94246fb6b7f3516
-$(package)_git_commit=18f4945d942cc53e336c40bf13080934179a9047
+$(package)_sha256_hash=2ec6ff7aa31e1482637a363772de7866d8f5f3712784ef60827b3ad725e8dbd7
+$(package)_git_commit=11552c1579656713495a34f102ba6cbeae8ea780
```

Comment:
This should be the merge commit `0af1ce8bf121e1ad367db907c39d214581e270a6`.

Reply:
Fixed